src/avram/queryable.cr

```diff
@@ -36,9 +36,19 @@ module Avram::Queryable(T)
       new.last?
     end
 
-    def self.truncate
+    # Removes all data from a table using the TRUNCATE postgres SQL command.
+    #
+    # You should not run this command without `cascade: true` if your table
```

Comment:
```suggestion
    # You should run this command with `cascade: true` if your table
```

Reply:
Happy to make the change. Would you mind explaining what the rule or rationale is for this?